src/main/frontend/components/views.cljs

```diff
@@ -1404,9 +1405,136 @@
 
     (table-view table option row-selection *scroller-ref)))
 
+(defn- get-views
+  [ent view-identity]
+  (let [entity (db/entity (:db/id ent))
+        views (->> (:logseq.property/_view-for entity)
+                   (filter (fn [view]
+                             (= view-identity (:logseq.property.view/identity view)))))]
+    (ldb/sort-by-order views)))
+
+(defn- create-view!
+  [view-parent view-identity]
+  (when-let [page (db/get-case-page common-config/views-page-name)]
+    (p/let [properties (cond->
+                        {:logseq.property/view-for (:db/id view-parent)
```

Comment:
Seems like it'd be helpful to have a View class for users to see and manage all their views. Happy to add that later if you think that'd be helpful

Reply:
Yes, a View class will be helpful to have.